packages/web-api/src/WebClient.spec.js

```diff
@@ -1612,6 +1612,46 @@ describe('WebClient', function () {
     });
   });
 
+  describe('has an option to suppress request error from Axios', () => {
```

Comment:
Thanks for adding these tests!

One thing we should do is ensure we mock out any calls to the slack.com API. Right now, since your tests are actually invoking e.g. `conversations.list()` method, this is issuing an HTTP request to slack.com/api. Not great, as we are a) unnecessarily creating traffic for Slack and b) have less control over the outcome of our tests. What if, during a test run, the Slack API goes down? Should the tests fail because of some service degradation? Probably not.

To handle this, we use the `nock` module to mock out API calls. Take a look at the nock setup and use on lines 150-175 of this file. Using `nock`, we can finely control a 'fake' response from the Slack API. This way, our tests will avoid both problems a) and b) above!

Additionally, since you have total control over the "Slack API" response using `nock`, you can not only test for the presence (and absence) of the `original` property in the test, you can even make sure that its contents are what you expect (since you control the API response).

Reply:
Ah! I understand. That does make sense.

I appreciate you taking the time to review this, share your thoughts, and explain the importance of each change. I apologize for any misunderstanding or oversight in my initial attempts.

Reply:
No worries at all! Zero judgment on our side, we appreciate you taking the time to send the PR and the improvements; the least we could do is help guide you along in the journey as we try to keep the quality and testing level of the project high working together 😄